libvast/src/system/exporter.cpp

```diff
@@ -384,15 +384,15 @@ behavior exporter(stateful_actor<exporter_state>* self, expression expr,
           self->send(x, atom::exporter_v, self);
     },
     [=](atom::run) {
-      VAST_INFO(self, "executes query:", to_string(self->state.expr));
+      VAST_VERBOSE(self, "executes query:", to_string(self->state.expr));
```

Comment:
I think we should print the `query_id` here and in all other log messages for the current query so we can actually know which query it is. Alternatively, we can change the name of the actor from `exporter` to `exporter-{query_id}`.

Reply:
We only get the query id in the `then()` part of the continuation, so we can't print it here.

Changing the actor name will break/be deprecated in CAF in the future afaik, the idea being is that an actor names should roughly correspond to a type name and always be constant.

Reply:
Alright, then just ignore this.